e2e/ansible/playbooks/hyperconverged/test-k8s-storage-pool/storage-pool-vars.yml

```diff
@@ -4,7 +4,7 @@ openebs_pvc: test-pvc.yaml
 
 openbs_storageclass: https://raw.githubusercontent.com/openebs/openebs/master/k8s/openebs-storageclasses.yaml
```

Comment:
These storage classes use the default storage pool. So, these changes will necessitate a custom storage class